from sinch.core.exceptions import SinchException


class AuthenticationException(SinchException):
    pass
